xbmc/windowing/android/WinSystemAndroidGLESContext.cpp

```diff
@@ -112,7 +112,11 @@ void CWinSystemAndroidGLESContext::PresentRenderImpl(bool rendered)
   if (!rendered)
     return;
 
-  m_pGLContext.SwapBuffers();
+  if (!m_pGLContext.TrySwapBuffers())
+  {
+    CEGLUtils::LogError("eglSwapBuffers failed");
+    throw std::runtime_error("eglSwapBuffers failed");
```

Comment:
pls. don't throw for android,

Reply:
Did we get problems on Android? Or just to be safe?

Reply:
We get them already (@da-anda knows)